packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -425,8 +427,12 @@ export class ExternalRequest<T extends Operation> {
    */
   async lookupRelations(tableId: string, row: Row) {
```

Comment:
Could be worth defining the type for this return - so it has a named type and its bit easier to follow, an array is a good choice.

Reply:
I refactored the returned type to be back to object, with a helper to get the key. This way we extract the logic from the consumer of this function